packages/v4/patternfly-docs/pages/componentGallery.css

```diff
@@ -0,0 +1,48 @@
+/* Toolbar styles */
+.ws-component-gallery > .pf-c-toolbar {
+  margin-left: calc(var(--pf-c-page__main-section--PaddingLeft) * -1);
+  border-bottom: var(--pf-c-page__sidebar--m-light--BorderRightWidth) solid var(--pf-c-page__sidebar--m-light--BorderRightColor);
+  margin-bottom: var(--pf-global--spacer--md);
+  width: calc(100% + var(--pf-c-page__main-section--PaddingLeft));
+  z-index: 101;
+}
+
+.ws-component-gallery > .pf-c-toolbar > .pf-c-toolbar__content {
+  --pf-c-toolbar__content--PaddingLeft: var(--pf-c-page__main-section--PaddingLeft);
+}
+
+/* Hide searchinput "x" button in favor of "Reset" text */
+.ws-component-gallery > .pf-c-toolbar .pf-c-text-input-group__utilities {
+  display: none;
+}
+
+/* Sidebar styles */
```

Comment:
This is impressive CSS! And it looks really good matching the design, working with what you have. That said, I'd love to use this and make some updates to the sidebar component.

* You should be able to have a divider between the sections. You can do that now manually, but you have to manage the orientation, it doesn't work with `.pf-m-panel-right` and doesn't work with a sticky panel in a stack layout (though I'd argue you shouldn't use a border there anyways).
* You should be able to add "body" elements to the sidebar panel and make those sticky so that the sidebar panel can remain full height and the things in it can be sticky.
* The body elements re: ^ should support padding and color variations - or somehow integrate with page sections better, as it's likely you have a sidebar as a full width layout like this and you want to recreate page sections (backgrounds, spacing/padding, etc).

I think that would go a long way here. I've added that to this issue - https://github.com/patternfly/patternfly/issues/5131